python/sglang/srt/layers/linear.py

```diff
@@ -162,14 +170,26 @@ def create_weights(
         layer.register_parameter("weight", weight)
         set_weight_attrs(weight, extra_weight_attrs)
 
+    def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
+        # Pack weight for get better performance on CPU
+        layer.weight = torch.nn.Parameter(
+            prepack_weight_if_needed(layer.weight.data),
+            requires_grad=False,
+        )
+
+        if layer.weight.device == torch.device("cpu") and cpu_has_amx_support():
+            self.fwd = sgl_kernel.cpu.weight_packed_linear
+        else:
+            self.fwd = F.linear
+
     def apply(
         self,
         layer: torch.nn.Module,
         x: torch.Tensor,
         bias: Optional[torch.Tensor] = None,
     ) -> torch.Tensor:
 
-        return F.linear(x, layer.weight, bias)
+        return self.fwd(x, layer.weight, bias)
```

Comment:
```python
if use_intel_amx_backend:
   return sgl_kernel.cpu.weight_packed_linear(x, layer.weight, bias)

return F.linear(x, layer.weight, bias)
```

Reply:
Fixed in https://github.com/mingfeima/sglang/pull/11/commits/1897f418c93408fa0e90d6e3a0bf0f69cb12a76a